projects/npm-tools/packages/npm-scripts/src/utils/deepMerge.js

```diff
@@ -14,17 +14,18 @@ const clone = (value, options) => merge(emptyTarget(value), value, options);
 function combineMerge(target, source, options) {
 	const destination = target.slice();
 
-	source.forEach((e, i) => {
+	source.forEach((element, i) => {
```

Comment:
`element` is a little odd to use here since we aren't talking about an html element. Could we rename this to `item`, which I think is generic enough for a "merge" function?